src/utils.ts

```diff
@@ -301,3 +301,31 @@ export function withFlickingMethods(prototype: any, flickingName: string) {
     };
   });
 }
+
+export function getBbox(element: HTMLElement, useOffset: boolean, includeXY: boolean) {
+  let bbox: BoundingBox;
+  if (useOffset) {
+    bbox = includeXY
```

Comment:
@WoodNeck 

The code is long. How about this?
```js
const x = includeXY ? ...
const y = includeXY ? ...
```

Reply:
Sometimes, I really am obsessed with such trivial things I think.
Actually I just wrote code like that to remove one single branch, which is `includeXY` in this case.
But why not, I think 2 `includeXY` is not bad.